### PR TITLE
Send empty location

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?222254

cc: @dannyroberts

### CODE REVIEW

File: corehq/apps/locations/tests/test_location_groups.py [hunk 1]
```diff
@@ -216,3 +216,44 @@ def test_location_fixture_generator(self):
...
+    @patch('corehq.apps.domain.models.Domain.uses_locations', lambda: True)
+    def test_location_fixture_generator_no_user_location(self):
+        """
+        Ensures that a user without a location will still receive an empty fixture
+        """
+        self.domain.commtrack_enabled = True
+        self.domain.save()
+        self.loc.delete()
+
+        state = make_loc(
```
Comment (dimagimon) [inline corehq/apps/locations/tests/test_location_groups.py]: F841 local variable 'state' is assigned to but never used

File: corehq/apps/locations/tests/test_location_groups.py [hunk 2]
```diff
@@ -216,3 +216,44 @@ def test_location_fixture_generator(self):
...
+
+    def test_location_fixture_generator_domain_no_locations(self):
+        """
+        Ensures that a domain that doesn't use locations doesn't send an empty
+        location fixture
+        """
+        self.domain.save()
+        self.loc.delete()
+
+        state = make_loc(
```
Comment (dimagimon) [inline corehq/apps/locations/tests/test_location_groups.py]: F841 local variable 'state' is assigned to but never used

File: corehq/apps/locations/tests/test_location_groups.py [hunk 3]
```diff
@@ -216,3 +216,34 @@ def test_location_fixture_generator(self):
...
+        """
+        Ensures that a domain that doesn't use locations doesn't send an empty
+        location fixture
+        """
+        self.domain.save()
+        self.loc.delete()
+
+        fixture = location_fixture_generator(self.user, '2.0')
+        try:
+            fixture[0]
```
Comment (benrudolph) [inline corehq/apps/locations/tests/test_location_groups.py]: nit: wouldn't `self.assertEqual(len(fixture), 0)` do the trick?

**Comment by czue** [comment]
:+1: from me. will give you a chance to respond to @benrudolph if you want

File: corehq/apps/locations/tests/test_location_groups.py (hunk 3 above)
Comment (emord) [inline corehq/apps/locations/tests/test_location_groups.py]: https://github.com/dimagi/commcare-hq/pull/10935/commits/3a21ecbd0d900fa521b8f663549453e4a7e95673

**Comment by emord** [comment]
:white_check_mark:

**Pull request merged**